src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -22,6 +22,10 @@ public static class SchemaVersionConstants
         public const int AddPrimaryKeyForResourceTable = (int)SchemaVersion.V25;
         public const int AddTextHashForStringSearchParameterVersion = (int)SchemaVersion.V26;
 
+        // Added TextHash column to StringSearchParameter and TokenStringCompositeSearchParam.
+        // SystemId, QuantityCodeId and RespurceTypeId nullable columns are marked as NOT NULL. Respective tables are back-filled with index 0 instead of null values
+        public const int AddPrimaryKeyPart2 = (int)SchemaVersion.V26;
```

Comment:
>AddPrimaryKeyPart2 [](http://example.com/codeflow?start=25&length=18)

you working in a same branch, no need to create new variables.

Reply:
I need to remove that top one

Reply:
I'm generally not a big fan of numbers in names.
Variables suppose to convey meaning to whose who reading code, and numbers are ... not that descriptive.
I personally would call it AddPrimaryKeyForSearchParamTables (kind a similar to v25), but final choice is up to you. But I would like to not have number in a variable. 

Also I think comment is a bit outdated, you do more than that it says.